src/PluginManager.js

```diff
@@ -79,11 +79,11 @@ export const regexRules = [
     },
         {
         type: "resource-target",
-        name: "resource-no-space-between-double-byte-kana-and-single-byte-character",
-        description: "Ensure that the target does not contain a space character between a double-byte kana and single-byte character.",
+        name: "resource-no-space-between-double-and-single-byte-character",
+        description: "Ensure that the target does not contain a space character between a double-byte and single-byte character.",
         note: "The space character is not allowed in the target string. Remove the space character.",
-        regexps: [ "([\\u3040-\\u309F]|[\\u30A0-\\u30FF]|[\\u4E00-\\u9FAF])\\s+[\\x00-\\xFF]|[\\x00-\\xFF]\\s+([\\u3040-\\u309F]|[\\u30A0-\\u30FF]|[\\u4E00-\\u9FAF])" ],
-        link: "https://github.com/ilib-js/i18nlint/blob/main/docs/resource-no-space-between-double-byte-kana-and-single-byte-character.md",
+        regexps: [ "([\\u3040-\\u309F\\u30A0-\\u30FF\\u4E00-\\u9FAF])\\s+[\\x00-\\xFF]|[\\x00-\\xFF]\\s+([\\u3040-\\u309F\\u30A0-\\u30FF\\u4E00-\\u9FAF])" ],
```

Comment:
There does not need to be parentheses around the character class, but it will work as is.